src/mon.c

```diff
@@ -2151,13 +2151,17 @@ register struct monst *mtmp;
         struct obj* list = collect_all_transient(NULL);
         struct obj* next;
         if (list) {
-            if (Blind && !Deaf)
-                You("hear a rush of objects jumble together nearby.");
-            else if (Blind && Deaf)
-                You_feel("a jarring vibration nearby.");
-            else
+            if (!Deaf) {
+                pline("A voice echoes in the arena:");
+                verbalize("You may choose one trophy from the spoils!");
```

Comment:
This doesn't match the diction used for other messages. In particular, it should prefer "thou" pronouns over "you". But "Thou mayest choose one trophy" is awkward; perhaps it should be imperative: "Choose thy trophy from the spoils!"